### PR TITLE
chore: added code of conduct

### PR DESCRIPTION
**Pull request by edgarvonk**
Added code of conduct. See: https://opensource.guide/code-of-conduct/

Solves PZ-3274

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (bas-info-nl) [inline CODE_OF_CONDUCT.md:63]:
```suggestion
[the ZAC maintainers](email:zaakafhandelcomponent@info.nl)
```
Or another email? And should we discuss this broader?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
 # Contribution guidelines
 
+We welcome contributions from the open source community.
 If you want to contribute, we ask you to follow these guidelines.
-These instructions are currently mainly intended for our own Lifely developers, but
-we do also welcome contributions from the open source community.
 Please reach out to us if you have questions or wish to contribute.
```
Comment (bas-info-nl) [inline CONTRIBUTING.md:5]:
```suggestion
Please reach out to [us](email:zaakafhandelcomponent@info.nl) if you have questions or wish to contribute.
```

Comment (bas-info-nl) [reply CONTRIBUTING.md:5]: Or leave it up to individuals to find a contributor's contact details?

**Review by bas-info-nl — APPROVED** [review]
Small comments on contact details

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (edgarvonk) [reply CODE_OF_CONDUCT.md:63]: Ah, missed that. :-) I would just use GitHub as contact method?

Funny that Maykin also missed this btw: https://github.com/open-zaak/open-zaak/blob/main/CODE_OF_CONDUCT.md ;-)

File: CONTRIBUTING.md (hunk 2 above)
Comment (edgarvonk) [reply CONTRIBUTING.md:5]: I would point them to https://github.com/infonl/dimpact-zaakafhandelcomponent/discussions and leave it at that.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (edgarvonk) [reply CODE_OF_CONDUCT.md:63]: Hmm, maybe email is better here because this is about sensitive stuff.

File: CONTRIBUTING.md (hunk 2 above)
Comment (edgarvonk) [reply CONTRIBUTING.md]: I did the latter now.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (edgarvonk) [reply CODE_OF_CONDUCT.md]: For now I did the latter: a mailto link to our #dimpact-team-genève-dev Slack channel.

**Pull request merged**